installing/installing_azure_stack_hub/installing-azure-stack-hub-default.adoc

```diff
@@ -0,0 +1,75 @@
+:_content-type: ASSEMBLY
+[id="installing-azure-stack-hub-default"]
+= Installing a cluster on Azure Stack Hub with an installer-provisioned infrastructure
+include::modules/common-attributes.adoc[]
+:context: installing-azure-stack-hub-default
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on Microsoft Azure Stack Hub with an installer-provisioned infrastructure. However, you must manually configure the `install-config.yaml` file to specify values that are specific to Azure Stack Hub.
+
+[NOTE]
+====
+While you can select `azure` when using the installation program to deploy a cluster using installer-provisioned infrastructure, this option is only supported for the Azure Public Cloud.
+====
+
+[id="prerequisites_installing-azure-stack-hub-default"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_azure_stack_hub/installing-azure-stack-hub-account.adoc#installing-azure-stack-hub-account[configured an Azure Stack Hub account] to host the cluster.
+* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
+* Installing the cluster requires that you download the RHCOS virtual hard disk (VHD) cluster image and upload it to your Azure Stack Hub environment so that it is accessible during deployment. Decompressing the VHD files requires approximiately 16 GB of local disk space.
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-azure-user-infra-uploading-rhcos.adoc[leveloffset=+1]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/installation-initializing-manual.adoc[leveloffset=+1]
+include::modules/installation-configuration-parameters.adoc[leveloffset=+2]
+include::modules/installation-azure-stack-hub-config-yaml.adoc[leveloffset=+2]
+
+include::modules/manually-create-identity-access-management.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
```

Comment:
Is this required for the `.Heading` version of these too? The guidelines aren't explicit and it's new enough that I don't know.

Reply:
Yep.